src/inhibitors/disabledGroup.js

```diff
@@ -7,8 +7,8 @@ module.exports = class extends Inhibitor {
     }
 
     async run(msg, cmd) {
-        if (!msg.guildSettings.disabledCommandsGroup.includes(cmd.category)) return;
-        throw msg.language.get("INHIBITOR_DISABLED_GROUP");
+        if (!msg.guildSettings.disabledCommandsGroup.includes(cmd.category.toLowerCase())) return;
+        throw msg.sendMessage(`${msg.language.get("INHIBITOR_DISABLED_GROUP")}: \`${cmd.category}\``);
```

Comment:
This is why languages support functions.
Please use i18n like it should be used.

Reply:
I intend language files do not have any functions so that people with no JS experience can also go around and translate them without messing up those strings.

Reply:
Thats not right. Language files should use functions. they're so helpful with organising. We're working on a way to support functions so this wont be a problem.